tests/console/system_prepare.pm

```diff
@@ -27,6 +27,13 @@ use warnings;
 
 sub run {
     my ($self) = @_;
+
+    if (check_var("BACKEND", "s390x") && check_var("UPGRADE", "1")) {
+        # Since we have issue bsc#1154156 so we have to switch to textmode
```

Comment:
if you need this as workaround to a bug then please

1. detect if the bug is actually present
2. `record_soft_failure` or use a workaround needle with the bug ref
3. apply the workaround only for affected product versions (or dynamically when bug was detected)

Reply:
> 
> 
> if you need this as workaround to a bug then please
> 
>     1. detect if the bug is actually present
> 
[GW]: you can check this result show vnc issue happen http://openqa.suse.de/tests/3945141#step/system_prepare/10
>     2. `record_soft_failure` or use a workaround needle with the bug ref
> 
[GW]: add record_soft_failure
>     3. apply the workaround only for affected product versions (or dynamically when bug was detected)
[GW]: add is_sle(15+) 
Verification is running now: 
Created job #3947462: sle-15-SP2-Migration-from-SLE11-SP4-to-SLE15-SP2-s390x-Build143.1-offline_sles11sp4_media_def_full_zVM_preparation@s390x-zVM-Upgrade-sp2 -> http://openqa.suse.de/t3947462
Created job #3947463: sle-15-SP2-Migration-from-SLE11-SP4-to-SLE15-SP2-s390x-Build143.1-offline_sles11sp4_media_def_full_zVM@s390x-zVM-Upgrade-sp2 -> http://openqa.suse.de/t394746

Reply:
>> if you need this as workaround to a bug then please
>> 1. detect if the bug is actually present
> you can check this result show vnc issue happen http://openqa.suse.de/tests/3945141#step/system_prepare/10

No, this is not what I meant. I mean the test code needs to detect the error condition so that the soft-fail is *only* applied if the bug actually happens.

Reply:
@okurz  updated, is that what you mean?

Reply:
No. You are not actually checking the behaviour in the product but only test variables. Just imagine that everything works as expected from product side, you would still end in the branch that records a soft failure. You need some checks, e.g. check with `wait_serial` or `assert_screen`+`match_has_tag` or `script_run`.

Reply:
@okurz  I have removed the work round part. Could you help check again?